.github/workflows/Build_and_publish_package.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Deploy Packages
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: RelWithDebInfo
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive   
+    - name: Build project
+      # Build project using powershell script
+      run: |
+        .\build.libgit2.ps1
+        .\buildpackage_forMendix.ps1 -version 1.101.1
+      shell: pwsh
```

Comment:
See comment above about `pwsh` vs `powershell` + let's add new lines between steps (before `- name` lines).

Reply:
updated all the references to use powershell instead of pwsh